src/ServiceControl/SagaAudit/SagaAuditComponent.cs

```diff
@@ -1,29 +1,18 @@
 ﻿namespace ServiceControl.SagaAudit
 {
-    using Microsoft.Extensions.DependencyInjection;
     using Microsoft.Extensions.Hosting;
-    using Operations;
     using Particular.ServiceControl;
     using ServiceBus.Management.Infrastructure.Settings;
 
     class SagaAuditComponent : ServiceControlComponent
```

Comment:
what is this still used for?

Reply:
We need this to call
> context.AddIndexAssembly(typeof(SagaSnapshot).Assembly);

This ensures RavenDB index setup for saga snapshots.